File: djJava/HelloJava02/src/studyjava02/Student.java
package studyjava02;

public class Student {
	public String name;
	public int age;
	public String gender;
	
	//생성자를 명시적으로 정의함
	//생성자에 매개변수를 넣을 수 있음
	//이렇게 생성자를 명시할 경우
	//기존에 암묵적으로 만들어졌던 빈 생성자는 사라지게 됨
	public Student(String name, int age, String gender) {
		//this는 클래스 자신을 의미함
		this.name=name;
		this.age=age;
		this.gender=gender;
	}

}

File: djCSharp/HelloCSharp02/HelloCSharp02/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
//using = c의 #include

namespace HelloCSharp02
{
    //public, get set 이런 부분들은 추후 설명 예정
    public class Student
    {
        //학번 : 절대 겹치지 않는 속성
        //이름, 성별, 나이 : 겹칠 수도 있는 속성
        public int Hakbeon { get; set; } //학번, 중복되면 안 됨
        public string Name { get; set; } //이름
        public string Gender { get; set; } //성별
        public int Age { get; set; } //나이
    }
}

